### PR TITLE
Add CfPs and other conferences from Pythondeadlin.es

### PR DESCRIPTION
**Pull request by JesperDramsch**
Hey folks, as mentioned in #89 I was thinking about feeding back my data from https://pythondeadlin.es ( https://gihub.com/JesperDramsch/python-deadlines )

Ran the Linter, and it passed. Here are all the changes:

Modifies:
- FOSDEM +CfP
- PyCon Ph +CfP
- Python Web Conf +CfP
- PyCascades +CfP
- Jupytercon +CfP
- PyCon Lithuania +CfP
- Moscow Conf +CfP
- Pycon IT + Sponsor
- Pydata London +Cfp +Sponsor
- PyCon Israel +CfP
- PyCon Russia +CfP
- PyCon Malaysia +Country
- PyCon TW +CfP
- PyCon CZ was Renamed from old name to PyCon Czechia
- PyCon India +Sponsor +CfP
- Python Brasil +CfP
- PyCon HK +Country
- PyData Tel Aviv +CfP

Added Conferences
- Pydata Seattle
- PyDay La Paz
- PyCon Poland
- PyCon MEA @GlobalDevSlam
- PyCon Slovakia 2024

All the best,
Jesper

### CODE REVIEW

File: 2023.csv [hunk 1]
```diff
@@ -1,50 +1,54 @@
...
 DjangoCon Australia,2023-08-18,2023-08-18,"Tarndanya (Adelaide), Australia",AUS,Adelaide Convention Centre,,2023-05-14,https://2023.djangocon.com.au,https://2023.pycon.org.au/program,https://2023.pycon.org.au/sponsor
 PyCon Australia,2023-08-18,2023-08-22,"Tarndanya (Adelaide), Australia",AUS,Adelaide Convention Centre,,2023-05-15,https://2023.pycon.org.au,https://2023.pycon.org.au/program,https://2023.pycon.org.au/sponsor
 PyCon Latin America,2023-08-24,2023-08-26,"Monterrey, Nuevo León, Mexico",MEX,,,2023-05-07,https://www.pylatam.org,https://www.papercall.io/pyconlatam23,https://www.pylatam.org/patrocinios/informacion
-PyCon Malaysia,2023-08-26,2023-08-26,,MYS,,,2023-05-30,https://pycon.my,https://www.papercall.io/pycon-my-2023,
-PyCon Taiwan,2023-09-02,2023-09-03,"Taipei, Northern Taiwan, Taiwan",TWN,,,,https://tw.pycon.org/2023,,https://tw.pycon.org/2023/en-us/sponsor
+PyCon Malaysia,2023-08-26,2023-08-26,Malaysia,MYS,,,2023-05-30,https://pycon.my,https://www.papercall.io/pycon-my-2023,
```
Comment (jonafato) [inline 2023.csv:34]:
```suggestion
PyCon Malaysia,2023-08-26,2023-08-26,,MYS,,,2023-05-30,https://pycon.my,https://www.papercall.io/pycon-my-2023,
```

File: 2023.csv [hunk 2]
```diff
@@ -1,50 +1,54 @@
...
-Python Brasil,2023-10-30,2023-11-05,"Serra Gaúcha, Rio Grande do Sul, Brazil",BRA,,,,https://2023.pythonbrasil.org.br,,https://2023.pythonbrasil.org.br/patrocinio_python_brasil_2023.pdf
+Python Brasil,2023-10-30,2023-11-05,"Serra Gaúcha, Rio Grande do Sul, Brazil",BRA,,,2023-06-08,https://2023.pythonbrasil.org.br,https://pretalx.com/python-brasil-2023/cfp,https://2023.pythonbrasil.org.br/patrocinio_python_brasil_2023.pdf
 DjangoCon Africa,2023-11-06,2023-11-11,"Zanzibar, Mjini Magharibi, Tanzania",TZA,,,,https://2023.djangocon.africa,https://2023.djangocon.africa/talks,https://2023.djangocon.africa/sponsor-us
-PyCon Hong Kong,2023-11-10,2023-11-11,,HKG,,,2023-06-18,https://pycon.hk/2023,https://pycon.hk/2023/pycon-hk-2023-cfp,
-PyData Tel Aviv,2023-11-14,2023-11-14,"Tel Aviv, Gush Dan, Israel",ISR,InterContinental David Tel Aviv,,,https://pydata.org/telaviv2023,https://pydata.org/telaviv2023/cfp,https://pydata.org/telaviv2023/sponsor
+PyCon Hong Kong,2023-11-10,2023-11-11,Hong Kong,HKG,,,2023-06-18,https://pycon.hk/2023,https://pycon.hk/2023/pycon-hk-2023-cfp,
```
Comment (jonafato) [inline 2023.csv:53]:
```suggestion
PyCon Hong Kong,2023-11-10,2023-11-11,,HKG,,,2023-06-18,https://pycon.hk/2023,https://pycon.hk/2023/pycon-hk-2023-cfp,
```

File: 2024.csv [hunk 3]
```diff
@@ -1,3 +1,4 @@
 Subject,Start Date,End Date,Location,Country,Venue,Tutorial Deadline,Talk Deadline,Website URL,Proposal URL,Sponsorship URL
+PyCon Slovakia,2024-03-15,2024-03-17,"Bratislava, Slovakia",,,,,https://2024.pycon.sk/,,
```
Comment (jonafato) [inline 2024.csv:2]:
```suggestion
PyCon Slovakia,2024-03-15,2024-03-17,"Bratislava, Slovakia",SVK,,,,https://2024.pycon.sk/,,
```

**Review by jonafato — COMMENTED** [review]
Thanks for these additional details and conference entries. A couple of inline change requests.
1. Added the country code for PyCon Slovakia
2. I think we should leave the "Location" field empty for conferences that don't have city / venue information posted. We already have country information captured in its own column, so leaving the location empty can serve as an indicator that more specific information is not yet available and serve as a reminder to fill it in when it's announced.

**Comment by JesperDramsch** [comment]
Thanks for the review and the corrections!

**Pull request merged**